packages/core/src/App/app.jsx

```diff
@@ -44,6 +45,16 @@ const AppWithoutTranslation = ({ root_store }) => {
     React.useEffect(() => {
         initializeTranslations();
 
+        Analytics.initialise({
+            growthbookKey: process.env.GROWTHBOOK_CLIENT_KEY,
+            growthbookDecryptionKey: process.env.GROWTHBOOK_DECRYPTION_KEY,
+            rudderstackKey:
+                process.env.NODE_ENV === 'production'
+                    ? process.env.RUDDERSTACK_PRODUCTION_KEY
+                    : process.env.RUDDERSTACK_STAGING_KEY,
```

Comment:
to not to play with magic NODE_ENV I found good approach with just window's hostname

`        rudderstackKey: ['.pages.dev', 'git-fork', 'localhost'].some((condition) =>
            window.location.hostname.includes(condition),
        )
            ? process.env.RUDDERSTACK_STAGING_KEY
            : process.env.RUDDERSTACK_PRODUCTION_KEY
            `
            
  if NODE_ENV works for you, please forget

Reply:
I think `NODE_ENV` works for me